XamarinCommunityToolkit/Views/MediaElement/iOS/MediaElementRenderer.ios.cs

```diff
@@ -191,10 +195,17 @@ TimeSpan Position
 		{
 			get
 			{
-				if (avPlayerViewController.Player.CurrentTime.IsInvalid)
-					return TimeSpan.Zero;
+				try
+				{
+					if (avPlayerViewController.Player.CurrentTime.IsInvalid)
+						return TimeSpan.Zero;
 
-				return TimeSpan.FromSeconds(avPlayerViewController.Player.CurrentTime.Seconds);
+					return TimeSpan.FromSeconds(avPlayerViewController.Player.CurrentTime.Seconds);
+				}
+				catch
+				{
+					return TimeSpan.Zero;
+				}
```

Comment:
Just this.
```csharp
    if (_avPlayerViewController.Player?.CurrentTime.IsInvalid ?? true)
        return TimeSpan.Zero;

    return TimeSpan.FromSeconds(_avPlayerViewController.Player.CurrentTime.Seconds);
```
We should try to find ways for avoiding null-refs rather than suppressing all exceptions.